core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -116,9 +124,17 @@ public static String getSymbol(String name, String title, String tooltip, String
         symbol = symbol.replaceAll("(<title>)[^&]*(</title>)", "$1$2");
         symbol = symbol.replaceAll("(class=\")[^&]*?(\")", "$1$2");
         symbol = symbol.replaceAll("(tooltip=\")[^&]*?(\")", "$1$2");
```

Comment:
This is inconsistent to line 93 and the test.
When a symbol loaded from the resource already contains a tooltip but no tooltip was set in the method call that would not be properly removed here.